collector/enqueue_server/enqueue_server.go

```diff
@@ -0,0 +1,126 @@
+package enqueue_server
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/SUSE/sap_host_exporter/collector"
+	"github.com/SUSE/sap_host_exporter/internal/sapcontrol"
+)
+
+func NewCollector(webService sapcontrol.WebService) (*EnqueueServerCollector, error) {
+
+	c := &EnqueueServerCollector{
+		collector.NewDefaultCollector("enqueue_server"),
+		webService,
+	}
+
+	c.SetDescriptor("owner_now", "TBD", nil)
```

Comment:
`TBD`?

Reply:
To Be Defined. Honestly, I don't know where to get the descriptions yet!

Reply:
ok please open a issue on github and we can follow-up. Also put the link of the issue as comment in code